README.md

```diff
@@ -12,6 +12,54 @@ An implementation of Mina protocol in Rust, with focus on web and Wasm compatibi
 
 ** As you can probably tell this is a WIP! Don't use for anything yet **
 
+## Building
```

Comment:
Awesome work on detailed specifications of how to build and run the overall project 🥇!